projects/netgrif-components-core/src/lib/configuration/configuration.service.ts

```diff
@@ -246,6 +259,30 @@ export abstract class ConfigurationService {
         return config.providers.auth.address + config.providers.auth.endpoints[endpointKey];
     }
 
+    /**
+     * Loads the configuration for the application by fetching data from the resource provider
+     * and initializes the application settings based on the fetched configuration.
+     *
+     * @return {Promise<any>} A promise that resolves with the fetched configuration or returns null
+     * if the configuration resolution is not enabled.
+     */
+    public loadConfiguration(): Promise<any> {
+        if (!this.APPLICATION_CONFIG.resolve_configuration) {
+            return null;
+        }
+        return this._resourceProvider.get$(
+                `/frontend-config/public/${this.APPLICATION_CONFIG.application}/${this.APPLICATION_CONFIG.type}`,
+                this.APPLICATION_CONFIG.gateway_url
```

Comment:
Consider implementing specific resource service for configuration domain and resolve this kind of urls inside there.